generator/resources/ec2_windows_nvidia_test_matrix.json

```
@@ -0,0 +1,32 @@
+[
+    {
+    "os": "win-11",
+    "instanceType":"g4dn.xlarge",
+    "ami": "cloudwatch-agent-integration-test-win-11*",
+    "family": "window"
```

```suggestion
    "family": "windows"
```
https://github.com/aws/amazon-cloudwatch-agent-test/blob/9bbdb9fbd0cad88f9ce2eb2329efbfa0f0998d6f/generator/resources/ec2_windows_performance_test_matrix.json#L42